plugins/inputs/lustre2/lustre2.go

```diff
@@ -526,6 +527,10 @@ func (l *Lustre2) getLustreProcBrwStats(fileglob string, wantedFields []*mapping
 
 		wholeFile, err := os.ReadFile(file)
 		if err != nil {
+			if strings.Contains(err.Error(), "permission denied") {
```

Comment:
I just realized that
```suggestion
			if errors.Is(err, os.ErrPermission) {
```
is much better and recommended. Could you please change?

Reply:
I agree, that's better! Tested working:
```
[root@host tmp]# sudo -u telegraf ./telegraf -config ./telegraf.conf  --debug
2024-03-26T17:21:44Z I! Loading config: ./telegraf.conf
2024-03-26T17:21:44Z I! Starting Telegraf 1.31.0-cebe5c92 brought to you by InfluxData the makers of InfluxDB
2024-03-26T17:21:44Z I! Available plugins: 233 inputs, 9 aggregators, 31 processors, 24 parsers, 60 outputs, 6 secret-stores
2024-03-26T17:21:44Z I! Loaded inputs: cpu disk diskio kernel lustre2 mem processes swap system
2024-03-26T17:21:44Z I! Loaded aggregators:
2024-03-26T17:21:44Z I! Loaded processors:
2024-03-26T17:21:44Z I! Loaded secretstores:
2024-03-26T17:21:44Z I! Loaded outputs: prometheus_client
2024-03-26T17:21:44Z I! Tags enabled:
2024-03-26T17:21:44Z I! [agent] Config: Interval:10s, Quiet:false, Hostname:"", Flush Interval:10s
2024-03-26T17:21:44Z D! [agent] Initializing plugins
2024-03-26T17:21:44Z D! [agent] Connecting outputs
2024-03-26T17:21:44Z D! [agent] Attempting connection to [outputs.prometheus_client]
2024-03-26T17:21:44Z I! [outputs.prometheus_client] Listening on http://[::]:9273/metrics
2024-03-26T17:21:44Z D! [agent] Successfully connected to outputs.prometheus_client
2024-03-26T17:21:44Z D! [agent] Starting service inputs
2024-03-26T17:21:50Z D! [inputs.disk] [SystemPS] => unable to get disk usage ("/sys/kernel/debug/tracing"): permission denied
2024-03-26T17:21:50Z D! [inputs.lustre2] open /proc/fs/lustre/osd-ldiskfs/MGS/brw_stats: permission denied
2024-03-26T17:21:50Z D! [inputs.lustre2] open /proc/fs/lustre/osd-ldiskfs/fs1-MDT0000/brw_stats: permission denied
2024-03-26T17:21:50Z D! [inputs.lustre2] open /proc/fs/lustre/osd-ldiskfs/fs1-OST0000/brw_stats: permission denied
2024-03-26T17:21:50Z D! [inputs.lustre2] open /proc/fs/lustre/osd-ldiskfs/fs1-OST0001/brw_stats: permission denied
```